### DISCUSSION TITLE
Select GPU for launch?

### DISCUSSION THREAD

**Discussion by bks5881** [cgisky1980/ai00_rwkv_server#62] [General]
I launched the model with the weights, but in the config file I was not able to find any place to set the GPU id. I have two GPU s and  no matter what i did, it always launched on GPU id 0. I wanted to launch it on GPU id 1. Can you suggest how to set it?

**Comment by cryscan** [comment]
In [`assets/configs/Config.toml`](assets/configs/Config.toml) set

```toml
[adapter]
Manual = 1
```

**Reply by cryscan** [reply]
Not necessarily be 1, you may want to try numbers 0, 1, 2... until the desired adapter is selected.
